K8s-deployment/Charts/catalogue/values.yaml

```diff
@@ -2309,15 +2410,18 @@ profanityCheckSdk:
     - name: profanity-check-config
       secret:
         secretName: profanity-check-config
-
+    - name: cat-tmp
+      emptyDir:
+        medium: Memory
 
   ## @param profanityCheckSdk.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the ProfanityCheckSdk container(s)
   ##
   extraVolumeMounts: 
     - name: profanity-check-config
       mountPath: /usr/share/app/config.json
       subPath: config.json
-  
+    - name: cat-tmp
```

Comment:
for this verticle/deployment ```profanity-check```
tmp  dir is not required. Its a python app and not a vertx app